hmy/api_backend.go

```diff
@@ -241,6 +240,12 @@ func (b *APIBackend) GetTransactionsHistory(address, txType, order string) ([]co
 	return hashes, err
 }
 
+// GetStakingTransactionsHistory returns list of staking transactions hashes of address.
+func (b *APIBackend) GetStakingTransactionsHistory(address, txType, order string) ([]common.Hash, error) {
+	hashes, err := b.hmy.nodeAPI.GetStakingTransactionsHistory(address, txType, order)
```

Comment:
might as well just `return b.hmy`.... the signatures are the same

Reply:
could you clarify your comment here?

Reply:
the signature of `b.hmy.nodeAPI.GetStakingTransactionsHistory` is the same as the RPC `GetStakingTransactionsHistory` so no point to hold onto intermediate variables

Reply:
i see. so no possibility they will/could become different between regular vs. staking txns, correct?

Reply:
of course its possible, its an open source project, anything can change. But  we can change when needed, rather than adds lines of code now.